libimgui-null-backend-test/manifest

```diff
@@ -0,0 +1,17 @@
+: 1
+name: libimgui-null-backend-test
+version: 1.86.0-a.0.z
+project: imgui
+summary: libimgui-null-backend-test for libimgui
+license: MIT
+description-file: README.md
+url: https://github.com/ocornut/imgui
+email: contact@dearimgui.com
+doc-url: https://github.com/ocornut/imgui/wiki
+src-url : https://github.com/ocornut/imgui
+package-url: https://github.com/build2-packaging/imgui
+package-email: swat.somebug@gmail.com
+
+depends: * build2 >= 0.15.0-
+depends: * bpkg >= 0.15.0-
```

Comment:
Just wondering, are you not missing the dependency for the`libimgui` package, i.e. `depends: libimgui == $`?

Reply:
I think this is taken care of the declaration in `libimgui/manifest` itself.
```
----------------
libimgui/manifest
----------------
tests: libimgui-null-backend-test == $
```

Individually initialising `libimgui-null-backend-test` shows that `libimgui` is also pulled in (at least locally).

So far my understanding with separated tests and examples has been that the main library's manifest has declarations as in the above snippet, and the tests, examples and benchmarks packages have an implicit `depends` on the main lib.